server/common/rest.py

```diff
@@ -333,29 +282,17 @@ def layout_obs_get(request, data_adaptor):
         )
 
 
+# TODO: I think this is no longer useful, but it is called. So just returning an effectively empty response for now
```

Comment:
Should we remove the call as well now or in another story perhaps? (affects FE)

Reply:
TODO: create a story to remove from FE call as well

Reply:
https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-explorer/214